microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -263,7 +263,7 @@ Developers can now use the experimental feature Web App URL Handlers in origin t
 
 Learn more about experimenting with URL handlers at [Experimental features in Progressive Web Apps (PWAs)](../how-to/handle-urls.md).
 
-Register your domain for the **Web App URL Handlers** trial at our [Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-url-handlers/registration/).
+Register your domain for the **Web App URL Handlers** trial at our [Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-url-handlers/registration/).<!-- todo: update per https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/ -->
```

Comment:
2
@captainbrosset @amandabaker - todo: update per https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/

Reply:
Same comment as above. We have two choices:

1. Either we keep What's New always up to date, in which case we need to delete the OT paragraph, and perhaps also add a note that says that URL Handlers don't exist anymore.
2. Or we agree that What's New pages are snapshots in time, and we don't change anything, apart maybe from adding a paragraph in the intro that says that things may fall out of date as new versions get released.

Reply:
result:
```
_2023 update: This feature is no longer in [Origin Trial](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/), so the following no longer applies, 
and the PWA URL Handlers feature was removed from the Chromium browser engine:_  
Register your domain for the **Web App URL Handlers** trial at our 
[Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-url-handlers/registration/).
```
and i will add Note at top of article